### PR TITLE
Executor messages to summit

### PR DESCRIPTION
**Pull request by CryptoMaxPlanck**
**Description**
Adds support for executor messages going to summit. Both base and manager.

### CODE REVIEW

File: agents/agents/executor/types/contract_type.go [hunk 1]
```diff
@@ -14,6 +14,8 @@ const (
 	LightInboxContract // LightInboxContract
 	// InboxContract is a TODO.
 	InboxContract // InboxContract
+	// SummitContract is a TODO.
```
Comment (trajan0x) [inline agents/agents/executor/types/contract_type.go:17]: Why is this a todo?

**Review by trajan0x — CHANGES_REQUESTED** [review]
Looks pretty good.

One thing I'm curious about is if anecdotally you guys have been seeing a higher % of errors after switching to anvil backend

File: agents/agents/executor/types/contract_type.go (hunk 1 above)
Comment (CryptoMaxPlanck) [reply agents/agents/executor/types/contract_type.go:17]: :p, let me fix these

**Comment by CryptoMaxPlanck** [comment]
and @trajan0x the error rate has been pretty consistent (which is not good, but not worse)

**Pull request merged**